src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/BlobEncoders.cs

```diff
@@ -74,24 +75,21 @@ public MethodSignatureEncoder PropertySignature(bool isInstanceProperty = false)
             return new MethodSignatureEncoder(Builder, hasVarArgs: false);
         }
 
-        public void CustomAttributeSignature(out FixedArgumentsEncoder fixedArguments, out CustomAttributeNamedArgumentsEncoder namedArguments)
+        [return: TupleElementNames(new[] { "FixedArguments", "NamedArguments" })]
```

Comment:
This seems correct, but be aware that the compiler does not generate such attribute (yet). @VSadov will work on that the week after next. 
If the serialization format needs to change for some compiler-motivated reason, this would have to be updated.

Aside, from that, I'm excited to see our first ValueTuple consumer! ;-)